includes/gateways/paypal/functions.php

```diff
@@ -287,7 +287,7 @@ function get_order_items( $purchase_data ) {
 				'currency_code' => edd_get_currency(),
 				'value'         => (string) edd_sanitize_amount( $item_amount ),
 			),
-			'discount'    => $item['discount'],
+			'discount'    => (string) edd_sanitize_amount( $item['discount'] ),
```

Comment:
I think we should revert this one--this is not passed through to PayPal; it's just added to the `$items` array for calculating and is `unset` in line 200.

Reply:
https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8881/commits/c23324cef4d62b752215377a523136f9e6b2d586